testsuite.zig

```diff
@@ -15,11 +15,13 @@ test "Get endpoint list" {
     }
 }
 
+const host = "tcpbin.com";
```

Comment:
Can you move this variable into the test and rename it `test_host`? It does not belong into the global namespace.

Reply:
[done](https://github.com/ikskuh/zig-network/pull/93/commits/b66001d3c55046f7c1449ff5e1f28fcee6546d54)